From 3bba470d438aec937549cd5b7d04510eeffcaad3 Mon Sep 17 00:00:00 2001
From: Dinolek <git@dinolek.me>
Date: Thu, 19 Jun 2025 21:16:58 +0200
Subject: [PATCH 028/115] dt-bindings: memory-controllers: mediatek,smi-common:
 Add support for MT6785

Add mediatek,mt6785-smi-common compatible with mediatek,mt6779-smi-common
fallback.

Signed-off-by: Dinolek <git@dinolek.me>
---
 .../bindings/memory-controllers/mediatek,smi-common.yaml     | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-common.yaml b/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-common.yaml
index 0762e0ff6..2b1106249 100644
--- a/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-common.yaml
+++ b/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-common.yaml
@@ -46,6 +46,11 @@ properties:
           - mediatek,mt8195-smi-sub-common
           - mediatek,mt8365-smi-common
 
+      - description: for mt6785
+        items:
+          - const: mediatek,mt6785-smi-common
+          - const: mediatek,mt6779-smi-common
+
       - description: for mt7623
         items:
           - const: mediatek,mt7623-smi-common
-- 
2.56.0

